src/SampleApp/Program.cs

```diff
@@ -117,17 +205,25 @@ static int Main(string[] args)
             _authKey = args[1];
             _secretKey = args[2];
             _advertisingToken = args[3];
+
+            //Note: If using the Integration environment (see https://unifiedid.com/docs/getting-started/gs-environments ), tokens can be generated using: https://example-srvonly-integ.uidapi.com/
+
             if (args.Length >= 5)
             {
                 _domain = args[4];
             }
 
-            ExampleBasicRefresh();
-            ExampleAutoRefresh();
-            ExampleSharing();
+            ExampleBidStreamClient(); //for DSPs
```

Comment:
can we say here:

_Disable either the ExampleBidStreamClient+ExampleAutoRefreshBidStreamClient or ExampleSharingClientcall here depending on the permission given to your API Key._

Reply:
next time